contributors.json

```diff
@@ -1,60 +1,68 @@
-    "discord": "Vicarious#0001",
-    "minecraft": "a961a5ca-571f-4b58-9232-b719eea57008",
-    "transifex": null,
-    "transifex_languages": [],
-    "other": []
-  },
-  {
-    "github": "Voknehzyr",
-    "discord": "Voknehzyr#9668",
-    "minecraft": "651a4d47-58d2-487c-859e-bc8b8dd6724a",
-    "transifex": "sdir01",
-    "transifex_languages": [ "ru_RU" ],
-    "other": []
-  },
-  {
-    "github": "JJBlue",
-    "discord": null,
-    "minecraft": "3ec01a75-1d2f-4082-9d89-420733faa6e3",
-    "transifex": null,
-    "transifex_languages": [],
-    "other": []
-  },
-  {
-    "_comment": "flixi_07",
-    "github": null,
-    "discord": null,
-    "minecraft": "e230f964-3a2b-44ba-89ab-7656f3037323",
-    "transifex": null,
-    "transifex_languages": [],
-    "other": [ "Reliable Bug catcher" ]
-  },
-  {
-    "github": null,
-    "discord": null,
-    "minecraft": "d38342fd-eebc-4a21-a6a7-2aaba2706f2f",
-    "transifex": "notBuiden",
-    "transifex_languages": [ "zh_CN" ],
-    "other": []
-  },
-  {
-    "_comment": "KagamineCyn",
-    "github": null,
-    "discord": null,
-    "minecraft": "45d0b795-8d3c-4558-9028-913d1c84ccb1",
-    "transifex": null,
-    "transifex_languages": [],
-    "other": [ "Modifier Inventor" ]
-  }
+[
+  {
+    "github": "Flo56958",
+    "discord": "Flo56958#0034",
+    "minecraft": "5b2c7922-ea02-4cfd-8003-e28778f8c225",
+    "transifex": "Flo56958",
+    "transifex_languages": [ "en_US", "de_DE" ],
+    "other": [ "MineTinker-Creator" ]
+  },
+  {
+    "github": "Draycia",
+    "discord": "Vicarious#0001",
+    "minecraft": "a961a5ca-571f-4b58-9232-b719eea57008",
+    "transifex": null,
+    "transifex_languages": [],
+    "other": []
+  },
+  {
+    "github": "Voknehzyr",
+    "discord": "Voknehzyr#9668",
+    "minecraft": "651a4d47-58d2-487c-859e-bc8b8dd6724a",
+    "transifex": "sdir01",
+    "transifex_languages": [ "ru_RU" ],
+    "other": []
+  },
+  {
+    "github": "JJBlue",
+    "discord": null,
+    "minecraft": "3ec01a75-1d2f-4082-9d89-420733faa6e3",
+    "transifex": null,
+    "transifex_languages": [],
+    "other": []
+  },
+  {
+    "_comment": "flixi_07",
+    "github": null,
+    "discord": null,
+    "minecraft": "e230f964-3a2b-44ba-89ab-7656f3037323",
+    "transifex": null,
+    "transifex_languages": [],
+    "other": [ "Reliable Bug catcher" ]
+  },
+  {
+    "github": null,
+    "discord": null,
+    "minecraft": "d38342fd-eebc-4a21-a6a7-2aaba2706f2f",
+    "transifex": "notBuiden",
+    "transifex_languages": [ "zh_CN" ],
+    "other": []
+  },
+  {
+    "_comment": "KagamineCyn",
+    "github": null,
+    "discord": null,
+    "minecraft": "45d0b795-8d3c-4558-9028-913d1c84ccb1",
+    "transifex": null,
+    "transifex_languages": [],
+    "other": [ "Modifier Inventor" ]
+  },
+  {
+    "github": "kaekaes",
+    "discord": "AtheroX#8624",
+    "minecraft": "99be3c12-049b-4196-9cf9-8312f88e1728",
+    "transifex": "kaekaes",
+    "transifex_languages": [ "en_US", "es_ES", "ca_ES" ],

```

Comment:
here should only be the languages listed you actively work on and that are available on transifex.

Reply:
Ok ^^